csfieldguide/package.json

```diff
@@ -13,11 +13,13 @@
     "sticky-state": "2.4.1",
     "iframe-resizer": "3.6.3",
     "csfg-module-fsa": "file:./static/js/modules/fsa/",
+    "csfg-interactive-algorithm-timer": "file:./static/interactives/algorithm-timer/",
     "csfg-interactive-big-number-calculator": "file:./static/interactives/big-number-calculator/",
     "csfg-interactive-bin-packing": "file:./static/interactives/bin-packing/",
```

Comment:
The bin packing interactive should un-display the 'congratulations' message if the bins are packed and the user presses 'unpack all'
![image](https://user-images.githubusercontent.com/25916475/56932484-24211800-6b38-11e9-875f-00791f7021d2.png)